src/components/icons/LaptopIcon.js

```diff
@@ -7,7 +7,8 @@ export default function LaptopIcon ({ className, color = '#000000' }) {
   return <Icon
     color={color}
     title={title}
-    className={className}>
+    className={className}
+  >
```

Comment:
I don't like this change. Is the linter making you do it? I don't think the version of standardjs we use makes us do this, but maybe a newer version does?

Reply:
Updated (... or downgraded? :joy: ) to 13.1.0. :+1: